RoddyCore/src/de/dkfz/roddy/Roddy.java

```diff
@@ -820,7 +820,9 @@ public static File getCompressedAnalysisToolsOverviewFile() {
     }
 
     public static List<File> getConfigurationDirectories() {
-        return loadFolderListFromConfiguration(RoddyStartupOptions.configurationDirectories, Constants.APP_PROPERTY_CONFIGURATION_DIRECTORIES);
+        List<File> list = loadFolderListFromConfiguration(RoddyStartupOptions.configurationDirectories, Constants.APP_PROPERTY_CONFIGURATION_DIRECTORIES);
+        list.add(getFolderForConfigurationFreeMode());
```

Comment:
What do you mean exactly? Files are identified by an id and Roddy will tell you, when there's a duplicate.